grails-app/views/contestApplication/showModules/mainContent/_contestApplicationDataMobileCallToAction.gsp

```diff
@@ -0,0 +1,27 @@
+<g:set var="callTitleMsg"
+       value="${g.message(code: 'contestApplication.callToAction.' + contestApplication.campaignStatusRSDTO + '.title')}"/>
+<g:set var="callSubtitleMsg"
+       value="${g.message(code: 'contestApplication.callToAction.' + contestApplication.campaignStatusRSDTO + '.subtitle')}"/>
+<g:set var="callButtonMsg"
+       value="${g.message(code: 'contestApplication.callToAction.' + contestApplication.campaignStatusRSDTO + '.button')}"/>
+<g:if test="${contestApplication.published && [org.kuorum.rest.model.communication.contest.ContestStatusDTO.VOTING, org.kuorum.rest.model.communication.contest.ContestStatusDTO.RESULTS].contains(contestApplication.contest.status)}">
```

Comment:
I think this if, will be affect to the whole div to show or not the button depending on the contest state